config/default.yml

```diff
@@ -453,7 +453,7 @@ RSpec/Focus:
   Description: Checks if examples are focused.
   Enabled: true
   VersionAdded: '1.5'
-  VersionChanged: '2.1'
+  VersionChanged: '<<next>>'
```

Comment:
There are no changes to the configuration options, so no changes here are necessary. Could you please remove it?